packages/gatsby-transformer-mdx-introspection/README.md

```diff
@@ -40,31 +47,122 @@ query MyQuery {
 }
 ```
 
-Please not the lowercase `apitype` in the query - the names are normalized to lowercase by the DOM parser used.
-
-Example response:
+### Example response:
 
 ```json
 {
   "data": {
     "allComponentInMdx": {
       "nodes": [
         {
+          "component": "ApiType",
           "attributes": [
             {
-              "name": "apikey",
+              "name": "apiKey",
               "value": "test"
             },
             {
               "name": "type",
               "value": "OutOfOrderPropertiesTestType"
             }
           ],
-          "page": {
-            "fields": {
-              "slug": "/api/types/"
+          "mdx": {
+            "file": {
+              "relativePath": "api/types/index.md"
             }
           }
         },
-...
+        ...
+      ]
+    }
+  }
+}
+```
+
+## Options
+
+**`lowercaseIdentifiers`** [boolean] (optional, defaults to false)
+
+Whether to have the `component` field (and `attribute.name`) be all lower case (legacy)
+
+**`trimWhitespace`** [boolean] (optional, defaults to true)
+
+Whether to trim leading/trailing whitespace in JSX snippets and string literals
+
+**`collapseWhitespace`** [boolean] (optional, defaults to true)
```

Comment:
same here?  Did you have own use cases for not collapsing the whitespace?

Reply:
I suggested combining these two, just because the default behavior might be undesired in certain circumstances. I don't have a use case myself for it, though.